### DISCUSSION TITLE
More Monsoon downtime - issues with XCS

### DISCUSSION THREAD

**Discussion by mo-roger-milton** [MetOffice/simulation-systems#211] [Monsoon]
Power issues affecting the storage arrays for XCS (the supercomputer that hosts Monsoon), over the weekend means that Monsoon is currently unavailable. The main HPC has been restored for use, but xcslc0 and xcslc1 (collaboration login nodes) remain down. We hope to restore these to service within the next hour.
 
I'll post an update here again once login is possible.
 
Once again, apologies for any disruption caused.

**Comment by mo-mahammedk** [comment]
The XCS login nodes for Monsoon users are now back and accessible again (as of 16:30 yesterday evening). Please note however that this is through one of a pair of switches, so we are running with reduced resiliency and some further interruption to service may be required to return to full resiliency.
 
Once again, apologies for the lack of service, especially following on from last week's outage which was completely unrelated.
